We do not need to check sources, docs, coverage, typing for this project.

--- jaraco_functools-4.6.0/tox.ini.orig
+++ jaraco_functools-4.6.0/tox.ini
@@ -8,10 +8,7 @@
 usedevelop = True
 extras =
 	test
-	check
-	cover
 	enabler
-	type
 passenv =
 	GITHUB_ACTIONS
 
